consensus/polybft/blockchain_wrapper.go

```diff
@@ -86,13 +91,17 @@ func (p *blockchainWrapper) ProcessBlock(parent *types.Header, block *types.Bloc
 	// TODO: Call validate block in polybft
```

Comment:
What about this TODO? Could it be removed?

Reply:
This is a leftover from v3 code.